src/containers/PageContainer/PageContainer.constants.ts

```diff
@@ -5,4 +5,16 @@ export enum Modal {
 	Wallet = 'Wallet',
 }
 
-export const MODAL_VISIBILITY_WINDOW_MIN_WIDTH = 1000;
+export const MODAL_VISIBILITY_WINDOW_MIN_WIDTH = 744;
```

Comment:
Updated to a lower width due to designs requiring 'Connect' button being displayed. Gave this a quick test and Modals still fit within this value width.